addon/templates/components/frost-object-browser.hbs

```diff
@@ -6,18 +6,13 @@
     {{/yield-slot}}
   </div>
   <div class="top-buttons" background-color='red'>
-    {{frost-button
-      icon='frost/infobar-create'
-      onClick=(action 'onCreate')
-      priority='tertiary'
-      size='medium'
-      text='Create'
-      vertical=true
-    }}
+    {{#yield-slot 'app-level-actions'}}
+      {{yield (block-params (action 'onCreate'))}}
+    {{/yield-slot}}
   </div>
 </div>
 <div class="body">
-  {{#yield-slot 'filters'}}
+  {{#yield-slot 'filter-pane'}}
```

Comment:
Why the rename, I personally like `filters`. I think these slots should have one word, simple names to make it easier to remember for the consumer.